pom.xml

```diff
@@ -11,20 +11,18 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.15.RELEASE</version>
+		<version>2.1.0.BUILD-SNAPSHOT</version>
```

Comment:
could we use `2.1.0.M2` here or do we need something from snapshot?

Reply:
M2 should be fine